README.md

```diff
@@ -14,56 +14,189 @@
 
 ## Key features
 
-- built using [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform)
-  and [Tiamat](https://github.com/ComposeGears/Tiamat) navigation library
-- support SVG/XML
-- convenient code formatting for generated icon
-  - remove redundant code
-  - remove unused imports 
-  - skip default ImageVector parameters
-- support drag and drop inside IDE
+- Support conversion from SVG and XML
+- Streamlined code formatting for generated icon:
+  * remove redundant code (e.g. `public` keyword)
+  * remove unused imports
+  * skip default ImageVector parameters
+  * support generation as [backing field or with lazy delegate](https://github.com/ComposeGears/Valkyrie/tree/task/update-readme?tab=readme-ov-file#export-formats)
```

Comment:
I would suggest rename all `Lazy delegate property` or `Lazy delegate` to `Lazy property`, that should be short and clear enought.

Reply:
updated